docs/content/en/docs/reference/clusterspec/baremetal.md

```diff
@@ -12,13 +12,577 @@ The following additional optional configuration can also be included:
 * [CNI]({{< relref "optional/cni.md" >}})
 * [multus]({{< relref "optional/multus.md" >}})
 
+To generate your own cluster configuration, follow instructions from the Bare Metal [Create production cluster]({{< relref "../../getting-started/production-environment/" >}}) section and modify it using descriptions below.
+For information on how to add cluster configuration settings to this file for advanced node configuration, see [Advanced Bare Metal cluster configuration]({{< relref "#advanced-bare-metal-cluster-configuration" >}}).
+
 
 ```yaml
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: Cluster
+metadata:
+  name: my-cluster-name
+spec:
+  clusterNetwork:
+    cniConfig:
+      cilium: {}
+    pods:
+      cidrBlocks:
+      - 192.168.0.0/16
+    services:
+      cidrBlocks:
+      - 10.96.0.0/12
+  controlPlaneConfiguration:              
+    count: 1
+    endpoint:
+      host: "<Control Plane Endpoint IP>"
+    machineGroupRef:
+      kind: TinkerbellMachineConfig
+      name: my-cluster-name-cp
+  datacenterRef:
+    kind: TinkerbellDatacenterConfig
+    name: my-cluster-name
+  kubernetesVersion: "1.22"
+  managementCluster:
+    name: my-cluster-name
+  workerNodeGroupConfigurations:
+  - count: 1
+    machineGroupRef:
+      kind: TinkerbellMachineConfig
+      name: my-cluster-name
+    name: md-0
+
+---
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: TinkerbellDatacenterConfig
+metadata:
+  name: my-cluster-name
+spec:
+  tinkerbellIP: "<Tinkerbell IP>"
```

Comment:
At the end of cluster creation process, once the target workload cluster is ready, the Tinkerbell infrastructure services stack is moved from the bootstrap kind cluster to the target cluster. This tinkerbellIP field above would expect a VIP that the user specifies that is a unique IP in their network range and does not conflict with other IPs. This is the VIP on which the Tinkerbell services will be exposed once on the target cluster, so that this stack can be used for future provisioning needs. When separate workload/management cluster is supported for Bare Metal, this will become a necessity.

Reply:
Good information! I'll add that.

Reply:
Done